src/App.css

```diff
@@ -1,104 +1,913 @@
-body {
+/* Global styles */
+* {
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
+}
+
+body {
   font-family: Arial, sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
 }
 
-.container {
+/* App container styles */
+.app-container {
```

Comment:
Great job @ONJoseph 🤝

- [ ] I can see you have done an amazing job, however just like the previous reviewer suggested, your design does not match the design given. Kindly make your design a close replica of what was given in the project requirements.
- Your design
![your-design](https://user-images.githubusercontent.com/63917725/215307808-25a73f50-61a1-4357-9603-fc535d3e4bdc.PNG)
- Given design
![given-design](https://user-images.githubusercontent.com/63917725/215307824-bf96bcff-c182-46a1-8d4b-eee347332f40.PNG)

Reply:
Hi @1GORDON That was my last submission chance! And time for submission is almost over. Can you please approve, while I work on improving the design? I spent my whole night debugging the dependency issues. Now, How do I even submit? Please advice? Thanks.

Reply:
Hi @1GORDON , While I appreciate your feedback, I spent the whole night debugging the dependency errors, got it fixed, and so tired managed to come up with this design. Moreover, this is my last submission chance, the third. How do I submit again now? Please kindly advice. Will appreciate if you can approve. Thanks.

Reply:
Hi @ONJoseph ✋

- [ ] Sorry about this. I understand what you have gone through. I can tell you I have been through the same. I noticed you had 2 extra reviews and since the design was still very lacking, I couldn't approve. I am still available for reviews. If you submit your work in the coming hour, I will be able to review it. Thank you for your reaching out